macros/provider.go

```diff
@@ -104,9 +104,9 @@ func (b *MacroProvider) populateRequestMacros(reqWrapper *openrtb_ext.RequestWra
 		}
 	}
 
-	userExt, err := reqWrapper.GetUserExt()
-	if err == nil && userExt != nil && userExt.GetConsent() != nil {
-		b.macros[MacroKeyConsent] = *userExt.GetConsent()
+	//is this happening after upconvert?
+	if reqWrapper.User != nil && len(reqWrapper.User.Consent) > 0 {
+		b.macros[MacroKeyConsent] = reqWrapper.User.Consent
```

Comment:
I'm not sure. I can reach out to the event framework author to see what their intent is. Depending on their answer we will either need to look at both the 2.5 and 2.6 locations or just the 2.6 location.